src/regridStates/gauIncr2native.nml

```diff
@@ -0,0 +1,31 @@
+&config
+ n_vars=4,                             ! number of vars in list below
+ variable_list(1)="soilt1_inc     ", 
+ variable_list(2)="soilt2_inc     ", 
+ variable_list(3)="slc1_inc     ", 
```

Comment:
Var list in one line might be better (personal preference to save space)
variable_list="soilt1_inc","slc1_inc","",

Reply:
Changed - see above.